index.rst

```diff
@@ -145,6 +147,7 @@ You may also directly use related links to see documents which match you the mos
 
    internal/ca_protocol
    internal/IOCInit
+   C/C++ API Reference <https://docs.epics-controls.org/projects/base/en/latest/libcom-api.html>
```

Comment:
This added link points to the page with links to all the LibCom headers, but I wouldn't call that *the* "C/C++ API Reference", its one of 4 pages that are now generated and listed on the Base index page under the heading "C/C++ Headers". The Database API headers should also be linked from here, but I'm happy for the menu and record-type header files to be omitted.